x/ibc/core/spec/01_concepts.md

```diff
@@ -7,6 +7,69 @@ order: 1
 > NOTE: if you are not familiar with the IBC terminology and concepts, please read
 this [document](https://github.com/cosmos/ics/blob/master/ibc/1_IBC_TERMINOLOGY.md) as prerequisite reading.
 
+## Client Creation, Updates, and Upgrades
+
+IBC clients are on chain light clients. The light client is responsible for verifying
+counterparty state. A light client can be created by any user submitting a client
+identifier and a valid initial `ClientState` and `ConsensusState`. The client identifier
+must not already be used. Clients are given a client identifier prefixed store to
+store their associated client state and consensus states. Consensus states are 
+stored using their associated height. 
+
+Clients can be updated by any user submitting a valid `Header`. The client state callback
+to `CheckHeaderAndUpdateState` is responsible for verifying the header against previously
+stored state. The function should also return the updated client state and consensus state 
+if the header is considered a valid update. A light client, such as Tendermint, may have
+client specific parameters like `TrustLevel` which must be considered valid in relation
+to the `Header`.
+
+Clients may be upgraded. The upgrade should be verified using `VerifyUpgrade`. It is not
```

Comment:
I think it's also relevant to mention how to upgrade the client if one of the chains halts/panics vs if the upgrade is planned

Reply:
This should go at client specific docs IMO since it could be implemented in a multitude of ways depending on the client

Reply:
ah yes. Although I think it's still relevant to add that as a comment (i.e that it is up to the light client implementation).